CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Fixes
 
 - Set client name with version in Android SDK ([#1274](https://github.com/getsentry/sentry-dart/pull/1274))
+- Implement required `loadStructuredBinaryData` method in `SentryAssetBundle`
```

Comment:
```suggestion
- Implement required `loadStructuredBinaryData` method in `SentryAssetBundle` ([#1272](https://github.com/getsentry/sentry-dart/pull/1272))
```

Otherwise CI will complain again 😅